CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Improved **update-release-notes** logs when changes in dependent API modules are detected.
 * Reverted changes released in version 1.19.0 in lint, lint will not fail on `demisto.results`, `return_outputs` and `LOG`.
 * Updated the **generate-docs** command to use the content graph instead of the id_set file.
+* **Validate** will now validate items which were edited in .pack-ignore.
```

Comment:
does it validate the files edited, or the whole pack?

Reply:
Only the specific files that their .pack-ignore section was edited.